metals/src/main/scala/scala/meta/internal/metals/doctor/HeadDoctor.scala

```diff
@@ -103,13 +116,33 @@ class HeadDoctor(
     }
   }
 
-  private def buildTargetsHtml(): String =
-    new HtmlBuilder()
-      .element("h1")(_.text(doctorTitle))
-      .call(buildTargetsTable)
-      .render
+  private def buildTargetsHtml(focusOnBuildTarget: Option[String]): String = {
+    doctors().map(_.buildTargetsJson())
+    val html =
+      new HtmlBuilder()
+        .element("h1")(_.text(doctorTitle))
+        .call(buildTargetsTable(_, focusOnBuildTarget))
+        .render
+    val script = focusOnBuildTarget.map(scrollToBuildTarget).getOrElse("")
 
-  private def buildTargetsTable(html: HtmlBuilder): Unit = {
+    script + html
+  }
+
+  private def scrollToBuildTarget(focusOnBuildTarget: String): String =
+    s"""|<script>
+        |  document.addEventListener("DOMContentLoaded", function() {
+        |    const header = document.getElementById('reports-${focusOnBuildTarget}');
+        |    if (header) {
+        |      header.scrollIntoView({ behavior: 'smooth' });
+        |    }
+        |  });
+        |</script>
+        |""".stripMargin
+
```

Comment:
I'm not sure about this. This requires us to enable scripts in this webview in VSCode, I'm not sure if there should be some safety concerns here.

Reply:
I think this is ok since we don't actually do import any dangerous scripts. And in the future we could add improvements to webview. Also, this is quite clever!